documentation/translating.rst

```diff
@@ -42,7 +42,7 @@ details.
      - :github:`GitHub <python/python-docs-hu>`,
        `mailing list <https://mail.python.org/pipermail/python-hu>`__
    * - `Indonesian (id) <https://docs.python.org/id/>`__
-     - Oon Arfiandwi (:github-user:`oonid`),
```

Comment:
For now please don't remove any names, we'd prefer that inactive co-ordinators remove themselves at present.

Reply:
Since the person has been unreachable despite multiple attempts, in my mind it is unlikely they will come back here and remove themselves. It would be more accurate to display the current active coordinators only.

Reply:
hello,
I have coordinated with @irvan-putra,
confirmed that he will lead the next translation effort.

Reply:
So is it enough that @oonid mentioned above or do I and @oonid need to create separate pull requests?